lightningd/log.c

```diff
@@ -18,6 +18,10 @@
 #include <sys/types.h>
 #include <unistd.h>
 
+#if DEVELOPER
+bool dev_no_backtrace;
```

Comment:
Maybe we could use [`RUNNING_ON_VALGRIND`](http://valgrind.org/docs/manual/manual-core-adv.html#manual-core-adv.clientreq) to disable when needed (with the downside of requiring `valgrind.h` during compile time).

Reply:
Yes, let's do that.  It's simpler, and we already have HAVE_VALGRIND_MEMCHECK_H...

Reply:
Uh, no... if they don't have the valgrind headers installed, but have valgrind, then "make check" will fail.  We need this :(